plugins/basic/nexus-repository-httpbridge/src/main/java/org/sonatype/nexus/repository/httpbridge/DefaultHttpResponseSender.java

```diff
@@ -48,7 +51,7 @@
     implements HttpResponseSender
 {
   @Override
-  public void send(final Response response, final HttpServletResponse httpResponse)
+  public void send(final @Nullable Request request, final Response response, final HttpServletResponse httpResponse)
```

Comment:
This impl only needs that, unsure if others may or may not which is why I suggested just pass in the request and leave it up to the impl to sort out what it needs.

FTR I also think we could make this non-nullable in the future, but for now I think this is sufficient.